lane.py

```diff
@@ -981,7 +1063,19 @@ def parentage(self):
                 raise ValueError("Lane parentage loop detected")
             seen.add(parent)
             yield parent
-    
+
+    @property
+    def depth(self):
+        """How deep is this lane in this site's hierarchy?
+        i.e. how many times do we have to follow .parent before we get None?
+        """
+        depth = 0
+        tmp = self
+        while tmp.parent:
+            depth += 1
+            tmp = tmp.parent
+        return depth    
+
```

Comment:
Would `len(self.parentage)` be a simpler way to write this?

Reply:
I think it would. I copied this from master without looking at it.

Reply:
Fair enough.